src/lib/externaldata.py

```diff
@@ -45,6 +45,7 @@
     "properties": {
         "type": {"type": "string"},
         "is-main-source": {"type": "boolean"},
+        "is-critical-source": {"type": "boolean"},
```

Comment:
I'm not a fan of using the term "critical", IMO it's too strong. Maybe "important" would fit better?

Reply:
I have no real opinion on this, indeed important can be used.

I wonder though, is there a better parameter name than `is-important-source`? Is it possible to somehow communicate the changed manifest editing/PR behaviour as part of the option?

e.g. `require-update-for-manifest-edits` or something? (But that one's a bit long).

Reply:
I think `is-important` would do.